lib/shrine/plugins/validate_virus_free.rb

```diff
@@ -19,7 +19,7 @@ def validate_virus_free(message: nil)
 
         def add_error_msg(message)
           if Rails.env.development? && message.match(/nodename nor servname provided/)
-            Shrine.logger.error('VIRUS SCANNING IS OFF. PLEASE START CLAMD')
+            Rails.logger.error('VIRUS SCANNING IS OFF. PLEASE START CLAMD')
```

Comment:
Shrine.logger in a Rails application simply delegates to `Rails.logger`, so we are changing it for simplicity and homogeneity.